src/App.jsx

```diff
@@ -1,35 +1,21 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// import './styles/app.scss'
+import { Routes, Route } from 'react-router-dom';
 
-function App() {
-  const [count, setCount] = useState(0)
+import Home from './routes/Home.jsx';
+import About from './routes/About.jsx';
+import Login from './routes/Login.jsx';
+import Profile from './routes/Profile.jsx';
+import NotMatch from './routes/NotMatch.jsx';
 
+const TodoApp = () => {
   return (
-    <div className="App">
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://reactjs.org" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </div>
-  )
-}
-
-export default App
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="about" element={<About />} />
+      <Route path="login" element={<Login />} />
+      <Route path="*" element={<NotMatch />} />
+      <Route path="profile" element={<Profile />} />
+    </Routes>
```

Comment:
- Kindly trace the routing components to confirm their functionality. 
![Capture](https://user-images.githubusercontent.com/26620878/230682775-5cc3fa4f-934e-4322-900c-7dfe8a71f43d.PNG)
-this is how the page currently looks on the dev server